docs/rest-server/API.md

```diff
@@ -440,10 +440,16 @@ Status: 200
     createdTime: "createdTimestamp",
     completedTime: "completedTimestamp",
     executionType: "executionType",
-    // Sum of succeededRetriedCount, transientNormalRetriedCount,
-    // transientConflictRetriedCount, nonTransientRetriedCount,
-    // and unKnownRetriedCount
-    retries: retriedCount,
+    // sum of retries
+    retries: retries,
+    retryDetails: {
+      // Job failed due to user or unknown error
+      user: userRetries,
+      // Job failed due to system/plaform error
+      system: systemRetries,
```

Comment:
"platform" is better than "system" due to:
1. More accurate, user job may also be a "system"
2. Our PAI is "platform" for AI, not "system" for AI
3. There are PaaS, but no "system" as a Service

so can you replace all "system" words with "platform" in this PR?

Reply:
I find that "system" is used in rest api document (https://github.com/Microsoft/pai/blob/master/docs/rest-server/API.md#post-token) multiple times. Shall we change them as well?

Reply:
Seems there is no need to distingush with user in that doc, so no ambiguity in the doc (the "system" in the doc implies PAI).
So, it is better to replace "in the system" to "in PAI", but it is not necessary, it depends on you :)

Reply:
fixed